src/components/loader/loader-square.test.tsx

```diff
@@ -79,7 +79,7 @@ test("when `size` prop is set to small, the expected width, height, and margin s
 
 // These styling tests required for coverage
 test("when inside button, the expected white background colour is applied", () => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the should be `true`, but as there is a `beforeEach` at the top of the file already you don't need to have any mock value here